roles/ansible_service_broker/tasks/install.yml

```diff
@@ -50,45 +51,60 @@
     resource_name: admin
     user: "system:serviceaccount:openshift-ansible-service-broker:asb"
 
+# Using oc_obj because oc_service doesn't seem to allow annotations
 - name: create ansible-service-broker service
-  oc_service:
+  oc_obj:
     name: asb
     namespace: openshift-ansible-service-broker
     state: present
-    labels:
-      app: openshift-ansible-service-broker
-      service: asb
-    ports:
-      - name: port-1338
-        port: 1338
-    selector:
-      app: openshift-ansible-service-broker
-      service: asb
-
-- name: create etcd service
-  oc_service:
-    name: etcd
-    namespace: openshift-ansible-service-broker
-    state: present
-    ports:
-      - name: etcd-advertise
-        port: 2379
-    selector:
-      app: openshift-ansible-service-broker
-      service: etcd
+    kind: Service
+    content:
+      path: /tmp/asbsvcout
+      data:
+        apiVersion: v1
+        kind: Service
+        metadata:
+          name: asb
+          labels:
+            app: openshift-ansible-service-broker
+            service: asb
+          annotations:
+            service.alpha.openshift.io/serving-cert-secret-name: asb-tls
+        spec:
+          ports:
+            - name: port-1338
+              port: 1338
+              targetPort: 1338
+              protocol: TCP
+          selector:
+            app: openshift-ansible-service-broker
+            service: asb
 
+# Using oc_obj because oc_route doesn't seem to allow labels
 - name: create route for ansible-service-broker service
-  oc_route:
+  oc_obj:
```

Comment:
I would actually prefer that we keep using `oc_route` since it will better handle any immutable fields being changed better than `oc_obj` can. We should updated `oc_route` to allow for the setting of labels